Toolset/libraries/revidelibrary.8.livecodescript

```diff
@@ -11629,6 +11633,14 @@ function revIDEObjectIsOpen pLongID
    return exists(pLongID)
 end revIDEObjectIsOpen
 
+local sLCBErrorCode
+constant kExtensionError = "extension: error occured with domain"
+private command revIDEInitialiseErrors
+   if sLCBErrorCode is empty then
+      put lineOffset(kExtensionError, the scriptExecutionErrors) into sLCBErrorCode
+   end if
+end revIDEInitialiseErrors
```

Comment:
I wonder if it would be adequate to just put the LCB error code from the C++ source code in as a constant.  As I understand it, the error codes aren't supposed to every change!